### DISCUSSION TITLE
可以让vscode的run和debug运行之前自动build一次吗

### DISCUSSION THREAD

**Discussion by owwkmidream** [xmake-io/xmake-vscode#142] [Q&A] [answered]
如题，就像Clion那样

**Comment by waruqi** [comment, marked as answer]
可以了

**Reply by owwkmidream** [reply]
感谢

**Comment by waruqi** [comment]
默认行为改动影响比较大，很多用户反馈过来问题，所以我改进下了

加了一个 buildRun 的命令，默认切回 runonly 了。。然后加了一个 配置切换 run mode ，可以自己切到 buildRun 默认。。

<img width="349" alt="image" src="https://user-images.githubusercontent.com/151335/192995275-70a594d8-87b6-43dd-a4d6-df6591602022.png">

**Reply by owwkmidream** [reply]
OK

**Reply by yeangPan** [reply]
切换之后F5好像还是提示  `Configuration have been changed, please rebuild program first!`
